sssd_test_framework/utils/tools.py

```diff
@@ -733,3 +743,66 @@ def __exec(self, cls, cmd: str, name: str | int, service: str | None = None) ->
             return None
 
         return cls.FromOutput(command.stdout)
+
+
+class SSHKeyUtils:
+    """
+    Interface to ssh-keygen command.
```

Comment:
Will this just serve as an interface to ssh-keygen or possibly other ssh key related functions as well later?  If it may be used for a wider purpose, I think the name fits.  If it is only for ssh-keygen, maybe the class name should be SSHKeyGen or SSHKeyGenUtils?

Same for the property above.   I'm not going to push so if this makes more sense as is, I'm ok with it.

Reply:
I'm not sure if we'll have more utilities for it, let's figure that out when/if we come to it?

Reply:
Sure.  Works for me.